app/src/main/java/com/example/boeing301house/Item.java

```diff
@@ -142,7 +142,7 @@ public String getDateString() {
         return dateString;
     }
 
-    public String getValueString(){return String.format("%.2f", value);}
+    public String getValueString(){return String.format(Locale.CANADA,"%.2f", value);}
```

Comment:
what is locale.CANADA

Reply:
specifies that inputs and stuff are like those that would be used in canada (english, etc.). IDE doesn't like it when you dont specify it so I just added it in.